apps/web/src/components/Member/index.tsx

```diff
@@ -0,0 +1,29 @@
+import Link from 'next/link';
+
+import { StyledMember, Card, ProfileImage, H1 } from './styles';
+
+interface MemberProps {
+  name: string;
+  username: string;
+}
+
+export const Member: React.FC<MemberProps> = ({ name, username }) => {
```

Comment:
This is correct but just for consistency, could you add the types like this?
```suggestion
export const Member = ({ name, username } : MemberProps) => {
```